packages/dbml-parse/src/lib/interpreter/utils.ts

```diff
@@ -112,4 +112,10 @@ export function isSameEndpoint(sym1: ColumnSymbol | ColumnSymbol[], sym2: Column
   const firstId = sym1.id;
   const secondId = (sym2 as ColumnSymbol).id;
   return firstId === secondId;
+}
+
+export function normalizeNoteContent(content: string): string {
+  const lines = content.split('\n');
+  const minIndent = Math.min(...lines.filter((line) => line.trim()).map((line) => line.match(/^\s+/)?.[0].length || 0));
```

Comment:
- Please split the filter and map result to a new line and assign them to some variables. This code is hard to read IMO. 
- Suggest change to make the logic to calculate number of leading whitespaces easier to read:
   ```js
   const numOfLeadingWs = line.length - line.trimStart().length;
   ```

Reply:
@nguyenalter  fixed anh